Remotion/ObjectBinding/Web/UI/Controls/Styles.cs

```diff
@@ -17,6 +17,7 @@
 using System;
 using System.ComponentModel;
 using System.Web.UI.WebControls;
+using Remotion.ObjectBinding.Web.UI.Controls.BocTextValueImplementation;
```

Comment:
I think this using is incorrect? It should either not be needed or if needed, the dependency is in the wrong namespace?